bayesian/utils.py

```diff
@@ -370,3 +374,65 @@ def get_request_count(rdb, external_request_id):
     count = rdb.session.query(StackAnalysisRequest).filter(\
                 StackAnalysisRequest.id == external_request_id).count()
     return count
+
+class GithubRead():
+    CLONED_DIR = "/tmp/stack-analyses-repo-folder"
+    PREFIX_GIT_URL = "https://github.com/"
+    PREFIX_URL = "https://api.github.com/repos/"
+    RAW_FIRST_URL = "https://raw.githubusercontent.com/"
+    MANIFEST_TYPES = ["pom.xml", "package.json", "requirements.txt"]
+
+    def get_manifest_files(self):
+        manifest_file_paths = []
+        for base, dirs, files in os.walk(self.CLONED_DIR):
+            if '.git' in dirs:
+                dirs.remove('.git')
+            if 'node_modules' in dirs:
+                dirs.remove('node_modules')
+            for filename in files:
+                if filename in self.MANIFEST_TYPES:
+                    filepath = base + '/' + filename
+                    manifest_file_paths.append({
+                        "filename": filename,
+                        "filepath": filepath
+                    })
+        return manifest_file_paths            
+
+    def get_files_github_url(self, github_url):
+        global CLONED_DIR
+        global PREFIX_URL
+        global RAW_FIRST_URL
+        global MANIFEST_TYPES
+        
+        manifest_data = []
+
+        self.del_temp_files()
+
+        repo_suffix = github_url.split("https://github.com/")[-1].rstrip()
```

Comment:
It's not that straightforward, please use [f8a_worker.utils.parse_gh_repo()](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/utils.py#L578)